server/src/librelay/relay_server.go

```diff
@@ -486,6 +475,8 @@ func (relay *RelayServer) CreateRelayTransaction(request RelayTransactionRequest
 		log.Println(err)
 		return
 	}
+	// Since the on chain calculation is too tight and sometime insufficient, we need to give some slack
```

Comment:
actually, the comment is not relevant, its not just adding slack, is properly calculating gas used.

Reply:
Fixed